packages/integration-tests/suites/tracing/browsertracing/long-tasks-enabled/test.ts

```diff
@@ -22,7 +22,7 @@ sentryTest('should capture long task.', async ({ browserName, getLocalTestPath,
   const [firstUISpan] = uiSpans || [];
   expect(firstUISpan).toEqual(
     expect.objectContaining({
-      op: 'ui.long-task',
+      op: 'ui.task',
```

Comment:
```suggestion
      op: 'ui.long-task',
```

For now I'm gonna revert this, because we rely on `long-task` specifically in the UI, once that has been changed we can update this.